### PR TITLE
[bootstrap] replace packages not available on Ubuntu 20.04

### PR DESCRIPTION
**Pull request by dsyx**
Use alternative packages:

```sh
$ sudo apt install bsdtar
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package bsdtar is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libarchive-tools

E: Package 'bsdtar' has no installation candidate
```

### CODE REVIEW

File: script/bootstrap [hunk 1]
```diff
@@ -24,15 +24,26 @@ die()
...
     sudo apt-get -y install libtool || die
     sudo apt-get -y install libglib2.0-dev || die
+
+    if [ "$PLATFORM" = "Ubuntu" -a $(echo "$RELEASE >= $UBUNTU2004" | bc) -eq 1 ]; then
+        echo "Ubuntu Release >= $UBUNTU2004"
+        # package bsdtar is not available, use libarchive-tools instead.
+        sudo apt-get -y install libarchive-tools || die
+    else
+        sudo apt-get -y install bsdtar || die
+    fi
```
Comment (bukepo) [inline script/bootstrap:46]: sudo apt-get -y install libarchive-tools || sudo apt-get -y install bsdtar || die

File: script/bootstrap [hunk 2]
```diff
@@ -24,15 +24,26 @@ die()
 
 install_packages_apt()
 {
+    PLATFORM=$(lsb_release -is)
+    RELEASE=$(lsb_release -rs)
+    UBUNTU2004=20.04
```
Comment (bukepo) [inline script/bootstrap:29]:
```suggestion
```

**Comment by dsyx** [comment]
:thumbsup: LGTM

**Review by bukepo — APPROVED** [review]
LGTM:+1:

**Pull request closed without merging**